src/components/NcAppSidebar/index.js

```diff
@@ -4,3 +4,4 @@
  */
 
 export { default } from './NcAppSidebar.vue'
+export { default as NcAppSidebarHeader } from './NcAppSidebarHeader.vue'
```

Comment:
As it is a public component, let's move it into a new `components/NcAppSidebarHeader` directory.

Otherwise this component needs to me imported with named import.